### DISCUSSION TITLE
Welcome to math-o-matic Discussions!

### DISCUSSION THREAD

**Discussion by logico-philosophical** [math-o-matic/math-o-matic#83] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

  For the maintainers, here are some tips for getting started with Discussions.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by dragonhailstone** [comment]
hi?

**Reply by logico-philosophical** [reply]
( ﾉ ﾟｰﾟ)ﾉ

**Comment by erados** [comment]
English.. (sigh)

**Reply by logico-philosophical** [reply]
\\ (ﾟｰﾟ \\ )
